corehq/form_processor/management/commands/run_submission_reprocessing_queue.py

```diff
@@ -1,61 +1,63 @@
 from __future__ import absolute_import
 from __future__ import unicode_literals
+
 from datetime import timedelta, datetime
+from time import sleep
 
-from django.conf import settings
+from django.core.management import BaseCommand
 from django.db.models import F
 from django.db.models import Q
 
+from corehq.form_processor.tasks import reprocess_submission
 from corehq.util.datadog.gauges import datadog_gauge
 from couchforms.models import UnfinishedSubmissionStub
-from hqscripts.generic_queue import GenericEnqueuingOperation, QueueItem
-from corehq.form_processor.tasks import reprocess_submission
+from dimagi.utils.logging import notify_exception
 
 ENQUEUING_TIMEOUT = 14 * 24 * 60    # 14 days (in minutes)
 
+BATCH_SIZE = 1000
+
 
 def _record_datadog_metrics():
     count = UnfinishedSubmissionStub.objects.count()
     datadog_gauge('commcare.submission_reprocessing.queue_size', count)
 
 
-class SubmissionReprocessingEnqueuingOperation(GenericEnqueuingOperation):
+class SubmissionReprocessingEnqueuingOperation(BaseCommand):
```

Comment:
Did you choose to keep the `SubmissionReprocessingEnqueuingOperation` subclass, which is extended by an empty `Command` class below, because its name explains what it does?

Reply:
Good question, there probably isn't a good reason to have it this way but I'm going to leave it for now.